deploy/ceph-conf.yaml

```diff
@@ -14,6 +14,10 @@ data:
     auth_cluster_required = cephx
     auth_service_required = cephx
     auth_client_required = cephx
+#   log_to_stderr = true  # uncomment to log command stderr in in csi-rbdplugin container logs
```

Comment:
This is one configmap for all the csi driver, we should make the comment generic not specific to rbd when its not required